summoner-cli/summoner.cabal

```diff
@@ -106,17 +111,17 @@ library
                      , neat-interpolation ^>= 0.3.2.2
                      , optparse-applicative ^>= 0.15
                      , process ^>= 1.6.1.0
-                     , relude ^>= 0.5.0
-                     , shellmet ^>= 0.0.1
+                     , relude >= 0.5.0 && < 0.7
```

Comment:
Shouldn't we always use `^>= 0.6.0.0` for relude? Like, is it okay to support multiple versions of `relude` like with `base` or should we always specify the only version to guard ourselves?

Reply:
We are building using stack with both versions of relude and having no issues, I think that it is okay, unless it brings troubles.